scripts/ninja.js

```diff
@@ -1369,13 +1392,13 @@ function sortFilesByDeps(domain, dependency_graph) {
   return result;
 }
 
-function updateRelease() {
+async function updateRelease() {
   runtimeNinja(false);
   stdlibNinja(false);
-  othersNinja(false);
+  await othersNinja(false);
```

Comment:
Why are we only awaiting othersNinja? runtimeNinja and stdlibNinja are async, too.

Reply:
Just doing step by step and see what's wrong.
I don't understand the build, what is intended to run in parallel, what is just random.
E.g. I was surprised to see that `jscomp/others/build.ninja` and `jscomp/others/release.ninja` are identical.
Not even sure why there is e distinction between build and release.